control/state.yaml

```diff
@@ -0,0 +1,390 @@
+openapi: 3.0.3
+info:
+  title: Control state models
+  description: >-
+    All control state schemas
+  version: ^0.0.0
+components:
+  schemas:
+    Control.State:
+      description: >-
+        Request for setting state of resources in traffic generator
+      type: object
+      required:
+        - choice
+      properties:
+        choice:
+          type: string
+          x-enum:
+            flow_transmit:
+              x-field-uid: 1
+            protocol:
+              x-field-uid: 2
+            route:
+              x-field-uid: 3
+            port_link:
+              x-field-uid: 4
+            port_capture:
+              x-field-uid: 5
+            lacp:
+              x-field-uid: 6
+          x-field-uid: 1
+        flow_transmit:
+          $ref: '#/components/schemas/Control.Flow.Transmit'
+          x-field-uid: 2
+        protocol:
+          $ref: '#/components/schemas/Control.Protocol'
+          x-field-uid: 3
+        route:
+          $ref: '#/components/schemas/Control.Route'
+          x-field-uid: 4
+        port_link:
+          $ref: '#/components/schemas/Control.Port.Link'
+          x-field-uid: 5
+        port_capture:
+          $ref: '#/components/schemas/Control.Port.Capture'
+          x-field-uid: 6
+        # lacp:
+        #   $ref: './lacp.yaml#/components/schemas/Lacp.State'
+        #   x-field-uid: 7
+    Link.State:
+      description: >-
+        Sets the link state of configured ports.
+      type: object
+      required: [state]
+      x-status:
+        status: deprecated
+        information: Please use `ControlPortLink` instead
```

Comment:
Should the information field not be identical for these and rest in this field as in api.yaml?
e.g. Please use `set_control_state` with `port_link` choice instead.

Reply:
Actually these are deprecation message for schema, which will come into effect when user initiates the schema object (and not when user is making an API call)